scenario/mocking.py

```diff
@@ -198,25 +197,20 @@ def _get_secret(self, id=None, label=None):
                 if canonicalize_id(s.id) == canonicalize_id(id)
             ]
             if not secrets:
-                raise SecretNotFoundError(id)
+                raise SecretNotFoundError(id) from None
```

Comment:
umh, not sure I understand what effect this has?

Reply:
None. No recollection of why I put that there, will remove.